omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/OrderTypeController1_10Test.java

```diff
@@ -162,7 +162,7 @@ public void shouldCreateANewOrderType() throws Exception {
 		assertEquals(orderType.get("description"), Util.getByPath(newOrder, "description"));
 	}
 	
-	@Test
+	@Test(expected = RuntimeException.class)
```

Comment:
@gitcliff  after your changes , now these tests return Exceptions ?? That very wrong

Reply:
@mozzy11  these tests were run against the update method in DelegatingCrudResource which was allowing reposting of changes metadata but the ticket description says this should not happen any more with metadata hence notifying the client with an exception ,,,thats why the tests that were permitiing the reposting of metadata data initially are now throwing exceptional errors

Reply:
@gitcliff  ,the tickect talks about forbidding re-posting metadata, not updating metadata. 
So idealy updating meta-data shouldnt have any problem

Reply:
Yes, @mozzy11  and @gitcliff I haven't looked deeply at the code but looking at the ticket, the idea is, for example, if you post to the Encounter endpoint, although you can change the encounter type of that encounter, you shouldn't be able to inadvertantly change any of the underlying properties (name, etc) of that encounter type. However, you should be able to post directly to the EncounterType endpoint and change the name, etc.

The idea is that you shouldn't be able to by mistake change a piece of metadata when updating some data that refers to that metadata.  Hope that makes sense!

Reply:
makes sense @mogoodrich

Reply:
@mozzy11 u can also have a look a[ this](https://talk.openmrs.org/t/should-metadata-be-updatetable-on-re-posting/25658) , it concurs with what @mogoodrich was explaining above